docs/src/scss/Global.module.scss

```diff
@@ -59,6 +59,10 @@
       color: var(--ifm-font-color-base);
     }
   }
+
+  .container article a:not(.hash-link) {
```

Comment:
I feel the not selector here is a bit too specific to a single scenario, being the right TOC. E.g. now tag names are also underlined which I feel doesn't need to be.

![afbeelding](https://github.com/pnp/cli-microsoft365/assets/38426621/0ede7a86-fc1e-4c5d-8c31-22fe067609c3)

The same goes for the edit page button. I'm sorry for falling back on this topic but maybe this change isn't needed. The change here would make it in a way compliant with WCAG compliant, more specifically, section [F73](https://www.w3.org/TR/2008/WD-WCAG20-TECHS-20081103/F73). After a bit more intensive research I came across the following discussion on [StackOverflow](https://stackoverflow.com/questions/71316954/is-removing-a-link-underline-on-hover-not-wcag-compliant) that really sheds light on this topic.

WCAG compliance ensures web content is accessible to people with disabilities. This provides rules for conformance, but also offers helpful advice. Following the advice is beneficial, but not mandatory for WCAG compliance. One non-normative recommendation, section F73, deals with visually evident links. Underlining links is common, but it depends on the webpage's design. When links are separate from text, using color alone may be enough. So, underlining links may not always be required for WCAG compliance. F73 recognizes that links within text may need specific attributes to be visually understood, but it doesn't automatically mean WCAG failure. Success depends on design choices and overall accessibility.

Reply:
Shouldn't this depend on the accent color? I think red is not the best pick 🤔 i suppose typically colorblind have problem with red and green 🤔. 
But ok I get it that this change becomes problematic 😉. Let's revert it and we may always try to improve in future 👍

Reply:
Colors shouldn't be a big problem overall. As long as there is sufficient contrast between the background and the text. https://webaim.org/articles/contrast/#sc141

Reply:
So to sum up what is like the desired action I should take 😅. Because I got a bit lost (too many new things on the build 😉).
Should we skip this for now and remove this change and not add underlines
Or
Should I try to work on this abut more correcting other links that got underlined but shouldn't?

Reply:
I would suggest that we leave this for the time being and revisit it at a later point 😄

Reply:
@Adam-it if this is also fine for you, then I can do this during the merge process. That way we can get all the open `docs-migration` PRs in 😉

Reply:
✅